curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa21971e3b743844849985.md

```diff
@@ -7,23 +7,23 @@ dashedName: step-151
 
 # --description--
 
-The player should hit if either `Math.random() > .2` **or** if the player's health is less than 20.
+The player should hit if either `Math.random() > .2` **or** if the player's health is less than `20``.
```

Comment:
All of the these changes look to be unrelated to the step about the shift method.
Plus, we already have a PR to address this issue.

Please revert all changes for this file back to its original state 👍